src/borg/archiver.py

```diff
@@ -2272,18 +2272,34 @@ def do_break_lock(self, args, repository):
 
                 # "sh:" pattern style is the default, so the following line is not needed:
                 P sh
-                R /
-                # can be rebuild
-                - /home/*/.cache
-                # they're downloads for a reason
-                - /home/*/Downloads
-                # susan is a nice person
-                # include susans home
-                + /home/susan
-                # don't backup the other home directories
-                - /home/*
-                # don't even look in /proc
-                ! /proc\n\n''')
+                # borg will consider only the files within a recursion root
+                # defined in this way: "R path/to/recursion_root"
+                R /home
```

Comment:
no leading slash in the text, but in the example.

Reply:
also, maybe better to understand: borg starts the recursion (discovering files to back up) from such a recursion root.

"consider ... within ..." sounds similar like a include pattern.